pkg/account/component_test.go

```diff
@@ -523,10 +513,10 @@ func TestUpdateAccountRevokeAccreditation(t *testing.T) {
 				assert.Equal(t, revokedAccred, kcUserRep.GetAttribute(constants.AttrbAccreditations)[0])
 				return nil
 			}).Times(1)
-		mocks.eventDBModule.EXPECT().ReportEvent(ctx, "UPDATE_ACCOUNT", "self-service", gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return(nil).Times(1)
+		mocks.auditEventsReporterModule.EXPECT().ReportEvent(ctx, gomock.Any()).Times(1)
```

Comment:
Don't need to specify .Times(1) (we used to write it but we are removing them now)